frontend/src/resources/resource.ts

```diff
@@ -10,6 +10,7 @@ export interface IResourceDefinition {
 }
 
 export interface IResource extends IResourceDefinition {
+    spec?: any
```

Comment:
Please remove this and instead set the type parameter correctly on line 150 of console/frontend/src/atoms.tsx